src/schema/show.graphql

```diff
@@ -19,8 +19,8 @@ scalar Date
 type Show {
   id: ID!
   title: String!
-  startTime: Date!
-  endTime: Date!
+  nextShowDate: Date!
+  duration: Int!
   djs: [DJ]!
```

Comment:
Do we need an array of social media as well?

Reply:
We do, but I'm gonna leave that for another PR. If you're interested in working on graphql, I'd recommend you take it!